### ISSUE TITLE
Version of sqlite3.dll

### ISSUE THREAD

**Issue by realli** [mapbox/mapbox-unity-sdk#1834]
We got "sqlite3.dll not found" error in some computer, and i know install .net framework 3.5 is a workaround, But we do not want to instruct our users to install.

And I just tried replace the dll with the sqlite.dll downloaded from https://sqlite.org/download.html. It just work. And I want to know which version sqlite3.dll you are using, so I can check if the replacing have other problem. Thanks

**Comment by liquidbuddha** [comment]
I would be very interested if this fixes the sqlite3.ddl issue. Right now I am also asking users to do the .net workaround. 

Do I understand that you simply replaced the sqlite3.dll in the CoreScripts folder with the one you DLed from sqlite.org? Did that create other issue for you?

**Comment by realli** [comment]
Yes, I replace the sqlite3.dll in the Core/Plugins/sqlite folder with the dll i downloaded from sqlite.org. I used the Precompiled Binaries for Windows one.  Seems like everything just work, but i am not sure.  MapBox maybe use the .net framework version, which need the .net framework been installed.

**Comment by liquidbuddha** [comment]
Excellent fix. Thank you! So far this is working for me as well. Very happy to be losing this dependency of .net 3.5
